EMF+AKS/interface/domestic_court.gui

```diff
@@ -0,0 +1,1073 @@
+			spriteType = "GFX_court_bg_council"
+			position = { x= 15 y = 178 }
+			Orientation = "UPPER_LEFT"
+		}
+		iconType = {
+			name ="court_bg_council_old" # Used if MnM is disabled
+			spriteType = "GFX_court_bg_council_old"
+			position = { x= 15 y = 178 }
+			Orientation = "UPPER_LEFT"
+		}
+		iconType = {
+			name ="repover_council_1"
+			spriteType = "GFX_repover_council_1"
+			position = { x= 16 y = 181 }
+		}
+		iconType = {
+			name ="repover_council_2"
+			spriteType = "GFX_repover_council_1"
+			position = { x= 16 y = 293 }
+		}
+		iconType = {
+			name ="repover_council_3"
+			spriteType = "GFX_repover_council_1"
+			position = { x= 16 y = 405 }
+		}
+		iconType = {
+			name ="repover_council_4"
+			spriteType = "GFX_repover_council_1"
+			position = { x= 16 y = 517 }
+		}
+		iconType = {
+			name ="repover_council_5"
+			spriteType = "GFX_repover_council_1"
+			position = { x= 16 y = 629 }
+		}
+		
+		listboxType = {
+			name = "councillor_box"
+			position = { x = 10 y = 199 }
+			size = { x=515 y=580 }
+			Orientation = "UPPER_LEFT"
+			spacing = 0
+			scrollbartype = "standardlistbox_slider"
+			borderSize = {x = 0 y = 0}
+		}
+	}
+		
+	windowType = {
+		name = "councillor_entry"
+		backGround =""
+		position = { x=0 y=0 }
+		size = { x=303 y=112 }
+		moveable = 0
+		dontRender = ""
+		horizontalBorder = "0"
+		verticalBorder = ""
+		fullScreen = no
+		
+		guiButtonType =	{
+			name ="portrait"
+			quadTextureSprite= "GFX_char_75"
+			position = { x= 31 y = 0 }
+			Orientation = "UPPER_LEFT"
+			pdx_tooltip = "CHARACTER_TOOLTIP"
+			pdx_tooltip_delayed = "CHARACTER_TOOLTIP_DELAYED"
+		}
+		guiButtonType = {
+			name ="portrait_frame"
+			quadTextureSprite = "GFX_charframe_75"
+			position = { x= 23 y = -9 }
+			Orientation = "UPPER_LEFT"
+		}
+		guiButtonType = {
+			name ="portrait_status"
+			spriteType = "GFX_overlay_char_status_75"
+			position = { x= 33 y = -4 }
+			Orientation = "UPPER_LEFT"
+		}
+		iconType = {
+			name ="portrait_relation"
+			spriteType = "GFX_overlay_char_relation_75"
+			position = { x= 83 y = -4 }
+			Orientation = "UPPER_LEFT"
+		}
+		iconType = {
+			name ="opinion_bg"
+			spriteType = "GFX_opinion_bg"
+			position = { x= 19 y = 63 }
+			Orientation = "UPPER_LEFT"
+		}
+		instantTextBoxType = {
+			name = "opinion"
+			position = { x = 11 y = 69 }
+			textureFile = ""
+			font = "Arial12"
+			borderSize = {x = 0 y = 0}
+			text = "UI_MISSING_TEXT"
+			maxWidth = 50
+			maxHeight = 18
+			fixedsize = yes
+			format = centre
+		}
+		
+		instantTextBoxType = {
+			name = "label"
+			position = { x = 130 y = -10 }
+			textureFile = ""
+			font = "vic_22"
+			borderSize = {x = 0 y = 0}
+			text = "UI_MISSING_TEXT"	
+			maxWidth = 220
+			maxHeight = 20
+			format = left
+			fixedsize = yes
+		}
+		instantTextBoxType = {
+			name = "name"
+			position = { x = 131 y = 17 }
+			textureFile = ""
+			font = "vic_18"
+			borderSize = {x = 0 y = 0}
+			text = "UI_MISSING_TEXT"	
+			maxWidth = 350

```

Comment:
@zijistark in EMF, file, this value is 250, not 350. only discrepancy between my result and yours.